modules/custom/social_eda/src/Types/Application.php

```diff
@@ -40,9 +40,13 @@ public static function fromId(string $id): self {
         'name' => 'Cron',
       ],
       'graphql' => [
-        'uuid' => '123e4567-e89b-12d3-a456-426614174001',
+        'uuid' => 'fd4b26f6-ec5e-47c2-8bf1-d8db7fc9d094',
```

Comment:
Can you please explain why you have changed this UUID?

Reply:
@ribel Honestly, for no important reason. Those are randomly generated UUIDs. I thought the first two were too similar, so I changed the second one.